tests/tensorflow/test_keras_pyfunc_model_works_with_all_input_types.py

```diff
@@ -355,6 +361,7 @@ def test_multi_multidim_input_model_spark_udf(
     np.testing.assert_allclose(actual, np.squeeze(expected), rtol=1e-5)
 
 
+# TODO: fix the custom_objects problem in model serving
```

Comment:
Can we file an issue and link to it? serving support is kinda strange with the Keras 3.0 coming out.

Reply:
Oh I just forgot to remove this comment, it's supported